fe/fe-core/src/main/java/com/starrocks/transaction/TransactionState.java

```diff
@@ -397,6 +396,10 @@ public void setTransactionStatus(TransactionStatus transactionStatus) {
         }
     }
 
+    public void notifyVisible() {
+        this.latch.countDown();
```

Comment:
From the old code, this should be countdown when transactionStatus == TransactionStatus.VISIBLE.

Reply:
Do you mean i should check `transactionStatus == TransactionStatus.VISIBLE` when I call the method `notifyVisible()` ?